modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -41,7 +41,7 @@ The Mule runtime engine passed functional testing against the following software
 | JDK | JDK 1.8.0, JDK 11
 |===
 
-This version of Mule runtime engine is bundled with the Runtime Manager agent plugin version 2.1.9.
+This version of Mule runtime engine is bundled with the Runtime Manager agent plugin version 2.4.21. For Runtime Manager Agent compatibility, see xref:release-notes::runtime-manager-agent/runtime-manager-agent-release-notes.adoc[Runtime Manager Agent Release Notes].
```

Comment:
version came from info in https://github.com/mulesoft/docs-release-notes/pull/4051/files